### PR TITLE
Quickstart 23: Remote access to EJBs

### PR DESCRIPTION
**Pull request by mmusgrov**
This is quickstart (authored by Jaikiran) showing how to remotely access stateful and stateless EJBS.

### CODE REVIEW

**Comment by sgilda** [comment]
Excellent README instructions! 

I tried this against both jboss-as-web-7.0.2.Final and jboss-eap-6.0.0.DR9. The server-side deploy is successful. However, the client "mvn exec:exec" results in a BUILD FAILURE in both cases:

[ERROR] Failed to execute goal on project jboss-as-quickstarts-ejb-remote-client: Could not resolve dependencies for project org.jboss.as.quickstarts:jboss-as-quickstarts-ejb-remote-client:jar:7.1.0.CR1-SNAPSHOT: Could not find artifact org.jboss.as.quickstarts:jboss-as-ejb-remote-server-side:jar:7.1.0.CR1-SNAPSHOT in jboss-public-repository-group (http://repository.jboss.org/nexus/content/groups/public/) -> [Help 1]

Am I missing something in my setup?

**Comment by mmusgrov** [comment]
Hi,

Sorry for the delay in responding (my cable modem has been playing up).

I forgot that the client needs to access the server side artifact. 
Instead of mvn package can you type

```
     mvn install
```

I will update the README.md accordingly.

Thanks for spotting this.

```
     Mike
```

**Comment by pmuir** [comment]
Mike, could you adjust the pull to exclude the bmt quickstart, so we can review and merge each separately...

File: ejb-remote/client/pom.xml [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+      <license>
+         <name>Apache License, Version 2.0</name>
+         <distribution>repo</distribution>
+         <url>http://www.apache.org/licenses/LICENSE-2.0.html</url>
+      </license>
+   </licenses>
+
+    <repositories>
+      <!-- Add the JBoss repository for org.jboss</groupId:jboss-ejb-client 
+           this will not be necessary if it has been uploaded to central -->
```
Comment (pmuir) [inline ejb-remote/client/pom.xml:24]: I will add this to the sync list

File: ejb-remote/client/src/main/resources/jboss-ejb-client.properties [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+remote.connections=default
+
+remote.connection.default.host=localhost
+remote.connection.default.port = 4447
+remote.connection.default.connect.options.org.xnio.Options.SASL_POLICY_NOANONYMOUS=false
+
+remote.connection.two.host=localhost
+remote.connection.two.port = 4447
+remote.connection.two.connect.options.org.xnio.Options.SASL_POLICY_NOANONYMOUS=false
```
Comment (pmuir) [inline ejb-remote/client/src/main/resources/jboss-ejb-client.properties:32]: Could we add some comments to this file?

**Comment by pmuir** [comment]
Minor comments only :-)

**Comment by mmusgrov** [comment]
Pete,

It is commit |749fdb6d 
https://github.com/mmusgrov/quickstart/commit/749fdb6dd407d3c8a39b5849e4b68d07c9bf1b98| 
that needs to be removed from pull request 
https://github.com/jbossas/quickstart/pull/33.

But the git hub pull request workflow doesn't seem to have any facility 
for doing this. How does one adjust a pull?

Mike

**Comment by pmuir** [comment]
You'll need to:

1) create a new branch (locally) that just contains the right commits
2) push that branch to your fork on github
3) create a new pull request from that branch
4) close the existing pull request.

I can help more on IRC or similar if needed :-)

**Comment by mmusgrov** [comment]
That is pretty much what I did before.

Perhaps at step 3) I will be offered the opportunity to pick which 
commits to include in the pull request.

Mike

**Comment by pmuir** [comment]
It's step 1 where you essentially make this selection.

**Comment by sgilda** [comment]
You are satisfied that the suggested changes have been made and it's 
ready to go?

**Comment by pmuir** [comment]
Not yet.

**Comment by mmusgrov** [comment]
The pull requests includes some changes from the BMT quickstart example. Closing and replacing with
https://github.com/jbossas/quickstart/pull/41

**Pull request closed without merging**